refarm_redirects/middleware.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+from django.contrib.redirects.models import Redirect
+from django.contrib.redirects.middleware \
+    import RedirectFallbackMiddleware \
+    as DjangoRedirectFallbackMiddleware
+from django.contrib.sites.shortcuts import get_current_site
+
+
+class RedirectFallbackMiddleware(DjangoRedirectFallbackMiddleware):
```

Comment:
`Fallback` is the wrong part of the class name here, because the PR removes that fallback-like logic in favor of precondition check an existence of redirects

Reply:
@artemiy312 , i'll change it to `RedirectAllMiddleware`, y

TODO